containers/Makefile

```diff
@@ -114,22 +114,21 @@ build/concent-api/concent-api.tar: build/repositories/concent/  build/virtualenv
 	mkdir --parents $(dir $@)
 	package-builder/build-concent-api.sh "build/repositories/concent/" "build/concent-api"
 
-
 build/nginx-proxy/concent-api-assets.tar: build/repositories/concent/ build/virtualenv/ build/package-builder/build-concent-api-assets.sh
 	mkdir --parents "$(dir $@)"
-	build/package-builder/build-concent-api-assets.sh \
-		"build/repositories/concent/"                 \
-		"$(abspath package-builder)"                  \
-		"$(abspath build)"                            \
-		"$(dir $@)"
+	build/package-builder/build-concent-api-assets.sh\
+	  "build/repositories/concent/"                  \
+	  "$(abspath package-builder)"                   \
+	  "$(abspath build)"                             \
+	  "$(dir $@)"
 
 build/nginx-proxy/concent-api-docs.tar: build/repositories/concent/ build/package-builder/build-concent-api-docs.sh
 	mkdir --parents $(dir $@)
-	build/package-builder/build-concent-api-docs.sh   \
-		"build/repositories/concent/"                 \
-		"$(abspath package-builder)"                  \
-		"$(abspath build)"                            \
-		"$(dir $@)"
+	build/package-builder/build-concent-api-docs.sh\
+	  "build/repositories/concent/"                \
```

Comment:
It's a good practice to always leave a space before `\`. Otherwise it's easy to run into problems like this:
``` bash
cat\
a.txt\
b.txt\
c.txt
```
```
bash: cata.txtb.txtc.txt: command not found
```

That's because bash does not add spaces on its own and sees your command as `cata.txtb.txtc.txt` rather than `cat a.txt b.txt c.txt`.

There is some whitespace at the beginning of the line in this case which would prevent this problem but that's not always the case so, as a rule, we always add at least one space at the end.

I've updated the issue for updating coding style to make it clear.